core/src/test/java/kafka/test/ClusterConfig.java

```diff
@@ -173,63 +199,104 @@ public static class Builder {
         private String listenerName;
         private File trustStoreFile;
         private MetadataVersion metadataVersion;
+        private Map<String, String> serverProperties = Collections.unmodifiableMap(new HashMap<>());
```

Comment:
Is `Collections.emptyMap();`  more suitable?

Reply:
Yes. The code here is redundant. Thanks